src/sentry/replays/usecases/replay_counts.py

```diff
@@ -62,6 +66,12 @@ def _get_replay_id_mappings(
     If select_column is replay_id, return an identity map of replay_id -> [replay_id].
     The keys of the returned dict are UUIDs, represented as 32 char hex strings (all '-'s stripped)
     """
+
+    if data_source == Dataset.Discover:
+        search_query_func = discover.query
+    elif data_source == Dataset.IssuePlatform:
+        search_query_func = issue_platform.query  # type: ignore[assignment]
```

Comment:
Just checking we actually write to this col from issue platform?

Reply:
yup https://github.com/getsentry/snuba/blob/8d4ea6917d5efef88fafe5ca33814dbf91d02595/snuba/datasets/processors/search_issues_processor.py#L203

Reply:
Ok - so as long as it's in the context for the event we pass to snuba it should work...

Are replays associated to transactions? If so, then perf issues from transactions should work automatically. Other issue platform types that make a "synthetic" event would need to explicitly include it there

Reply:
yup, replays are associated to transactions. its done via the dynamic sampling context, which then adds the replay context. and for others:

rage click issue: https://github.com/getsentry/sentry/blob/8666d0c9f60be94988545522e1ff74aa7c0002f5/src/sentry/replays/usecases/ingest/issue_creation.py#L90

feedback issue:
https://github.com/getsentry/sentry/blob/8666d0c9f60be94988545522e1ff74aa7c0002f5/src/sentry/feedback/usecases/create_feedback.py#L98

Crons
-- not applicable

that should be everything